src/test/java/com/google/cloud/genomics/dataflow/functions/AlleleSimilarityCalculatorTest.java

```diff
@@ -0,0 +1,84 @@
+package com.google.cloud.genomics.dataflow.functions;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.api.services.genomics.model.Variant;
+import com.google.cloud.dataflow.sdk.transforms.DoFnTester;
+import com.google.cloud.dataflow.sdk.values.KV;
+import com.google.cloud.genomics.dataflow.utils.DataUtils;
+
+public class AlleleSimilarityCalculatorTest {
+
+  static final Variant snp1 = DataUtils.makeVariant("chr7", 200019, 200020, "T", newArrayList("G"),
+      DataUtils.makeCall("het-alt sample", 1, 0), DataUtils.makeCall("hom-alt sample", 1, 1),
+      DataUtils.makeCall("hom-ref sample", 0, 0), DataUtils.makeCall("hom-nocall sample", -1, -1),
+      DataUtils.makeCall("ref-nocall sample", -1, 0));
+
+  static final Variant snp2 = DataUtils.makeVariant("chr7", 200020, 200021, "C", newArrayList("A"),
+      DataUtils.makeCall("hom-alt sample", 1, 1), DataUtils.makeCall("het-alt sample", 0, 1),
+      DataUtils.makeCall("ref-nocall sample", 0, -1));
+
+  @Before
+  public void setUp() throws Exception {}
+
+  @Test
+  public void testIsReferenceMajor() {
+    assertTrue(AlleleSimilarityCalculator.isReferenceMajor(snp1));
+    assertFalse(AlleleSimilarityCalculator.isReferenceMajor(snp2));
+  }
+
+  @Test
+  public void testGetSamplesWithVariant() {
+    assertEquals(4, AlleleSimilarityCalculator.getSamplesWithVariant(snp1).size());
+    assertEquals(3, AlleleSimilarityCalculator.getSamplesWithVariant(snp2).size());
+  }
+
+  @Test
+  public void testSharedMinorAllSimilarityFn() {
+
+    CallSimilarityCalculatorFactory fac = new SharedMinorAllelesCalculatorFactory();
+    DoFnTester<Variant, KV<KV<String, String>, KV<Double, Integer>>> simFn =
+        DoFnTester.of(new AlleleSimilarityCalculator(fac));
+
+    List<KV<KV<String, String>, KV<Double, Integer>>> outputSnp1 = simFn.processBatch(snp1);
+    assertEquals(6, outputSnp1.size());
+    Assert.assertThat(outputSnp1, CoreMatchers.hasItems(
```

Comment:
Why not statically import assertThat? You statically imported assertTrue, assertFalse, and assertEquals.